numpy/_core/src/multiarray/einsum.c.src

```diff
@@ -816,7 +816,7 @@ PyArray_EinsteinSum(char *subscripts, npy_intp nop,
 
     NpyIter *iter = NULL;
     sum_of_products_fn sop;
-    npy_intp fixed_strides[NPY_MAXARGS];
+    npy_intp *stride;
```

Comment:
This is unnecessarily overshadowed further down.  Please make sure to not call `NpyIter_GetInnerStrideArray` twice.

Reply:
This is addressed, thanks